### PR TITLE
NDRS-488: Update default configs in `/local/chainspec.toml`

### PR DESCRIPTION
**Pull request by fizyk20**
This PR updates the Highway config and adds a sanity check for the future, printing a warning if the values don't make sense.

### CODE REVIEW

**Review by goral09 — APPROVED** [review]
Nicely done.

**Review by afck — APPROVED** [review]

File: node/src/components/chainspec_loader/chainspec.rs [hunk 1]
```diff
@@ -111,6 +112,16 @@ impl Default for HighwayConfig {
     }
 }
 
+impl HighwayConfig {
+    /// Checks whether the values set in the config make sense and prints warnings if they don't
+    pub fn validate_config(&self) {
+        let min_era_ms = 1u64 << self.minimum_round_exponent;
+        if self.era_duration.millis() < self.minimum_era_height * min_era_ms {
+            warn!("Era duration is less than minimum era height * round length!");
+        }
```
[the remark this answers is not in the document]
Comment (fizyk20) [reply node/src/components/chainspec_loader/chainspec.rs:121]: Good idea!

[the remark this answers is not in the document]
Comment (afck) [reply node/src/components/chainspec_loader/chainspec.rs:125]: Great, thank you!

**Pull request merged**